packages/webui/src/components/ui/TraceDetail.tsx

```diff
@@ -20,8 +20,14 @@ import ResponseHeaders from './ResponseHeaders';
 import { TabContent, TabList, TabListItem } from './Tabs';
```

Comment:
I don't think it's related to these code changes, but it does look to be related to a new mock trace that had been added at some point:

![image](https://github.com/FormidableLabs/envy/assets/17857833/f2c4901f-6694-4b79-9b6f-cad2b0b1e4d1)

This is the mock GQL "PDP" data trace: https://github.com/FormidableLabs/envy/blob/9531b8b91990205630f406afe53ca9961fca8ca3/packages/webui/src/testing/mockTraces/large-gql.ts#L1

And if you look at the screenshot, it doesn't look like its parsing and presenting the query correctly.  Compare this with the mock trace the GQL "People" query:

![image](https://github.com/FormidableLabs/envy/assets/17857833/3c80975a-5492-4bf4-bc48-f94c4f9f1829)

Also, in this above example, the code display part should really be full width, even if the content is narrow.  It looks... odd... to make the box narrow like this IMO.